lib/librato/version_specifier.rb

```diff
@@ -0,0 +1,15 @@
+module VersionSpecifier
```

Comment:
If you made this a class instead you could dependency inject `Rails` by default as a config option and then make your tests better. For example:

```ruby
class VersionSpecifier

  def self.supported(opts={})
    new(::Rails).supported(opts)
  end
  
  def initialize(env)
    @env = env
  end
  
  def supported(opts={})
    return unless block_given?
    return if !opts.key?(:min) && !opts.key?(:max)
    return yield if version >= opts[:min].to_s && !opts.key?(:max)
    return yield if version <= opts[:max].to_s && !opts.key?(:min)
    return yield if version.between?(opts[:min].to_s, opts[:max].to_s)
  end
  
  private
  
  def self.version
    [@env::VERSION::MAJOR, @env::VERSION::MINOR].compact.join('.')
  end
end
```

Then you can DI your own env object to make your tests more explicit when testing against variable rails vers.